tickplate.js

```diff
@@ -2,13 +2,7 @@
 
 const tickplate =
   (strings, ...keys) =>
-  (values) => {
-    const result = [strings[0]];
-    for (let i = 0; i < keys.length; i++) {
-      const key = keys[i];
-      result.push(values[key], strings[i + 1]);
-    }
-    return result.join('');
-  };
+  (values) =>
+    String.raw(strings, ...keys.map((key) => values[key] ?? ''));
```

Comment:
- We support node.js 12, so can;t use nullish coalescing operator, see: https://node.green/#ES2020-features--nullish-coalescing-operator-----
- We'd prefer for loop here